client/dfclient/src/process_death_report.cpp

```diff
@@ -0,0 +1,449 @@
+/** 
```

Comment:
At first it was unclear to me that this is in fact part of the GameplayState class. Maybe we could rename this file to `gameplay_state_deaths.cpp` or something like this?

Reply:
Is `gameplay_state_death_reports.cpp` ok?

Reply:
yes